README.md

```diff
@@ -1,64 +1,52 @@
-# Project Name
+# Ruby Enumerable Project
 
-> Description the project.
+> Ruby Enumarable Project.
 
 ## Built With
 
 - Ruby
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+N/A
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
+- Simply clone the project (for more information please check the link https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository
+- Run the cmd promt
+- Type "npm install" for the dependencies
```

Comment:
- [OPTIONAL] Please kindly the update your `Readme file` file with steps to run your `Ruby app`, the above command is for `node apps`.

Reply:
Thank you, we will change that part. :)